docs/replication_modes.rst

```diff
@@ -40,38 +40,31 @@ Synchronous mode
 
-Synchronous mode can be switched on and off via Patroni REST interface. See :ref:`dynamic configuration <dynamic_configuration>` for instructions.
+You can ensure that a standby never becomes a synchronous standby by setting ``nosync`` tag to true. This is recommended to set for standbys that are behind slow network connections and would cause performance degradation when becoming a synchronous standby.
+
+Synchronous mode can be switched on and off using ``patronictl edit-config`` command. See :ref:`dynamic configuration <dynamic_configuration>` for instructions.
 
 
 Synchronous mode implementation
 -------------------------------
 
-When in synchronous mode Patroni maintains synchronization state in the DCS, containing the latest primary and current synchronous standby. This state is updated with strict ordering constraints to ensure the following invariants:
-
-- A node must be marked as the latest leader whenever it can accept write transactions. Patroni crashing or PostgreSQL not shutting down can cause violations of this invariant.
-
-- A node must be set as the synchronous standby in PostgreSQL as long as it is published as the synchronous standby.
-
-- A node that is not the leader or current synchronous standby is not allowed to promote itself automatically.
-
-Patroni will only ever assign one standby to ``synchronous_standby_names`` because with multiple candidates it is not possible to know which node was acting as synchronous during the failure.
+When in synchronous mode Patroni maintains synchronization state in the DCS, containing the latest primary, number of nodes required for quorum and nodes currently eligible to vote on quorum. In steady state the nodes voting on quorum are the leader and all synchronous standbys. This state is updated with strict ordering constraints with regards to node promotion and ``synchronous_standby_names`` to ensure that at all times any subset of voters that can achieve quorum is contained to have at least one node having the latest successful commit.
```

Comment:
> In steady state the nodes voting on quorum are the leader and all synchronous standbys.

Judging from [this line](https://github.com/zalando/patroni/pull/672/files#diff-a839e95c7271edf3b62934f9e0e61f16R131), steady state also means updates to quorum and/or replication factor have completed (as opposed to simply having `self.voters == self.sync`).